src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -483,7 +483,7 @@ ResultWithValue<FrameTimestamp> AudioStreamAAudio::getTimestamp(clockid_t clockI
         aaudio_result_t result = mLibLoader->stream_getTimestamp(stream, clockId,
                                                                  &frame.position,
                                                                  &frame.timestamp);
-        if (result == AAUDIO_OK){
+        if (result == (aaudio_result_t)Result::OK){
```

Comment:
`static_cast<aaudio_result_t>(Result::OK)` is preferred to raw cast

Reply:
Done